src/PhpBrew/Patches/Apache2ModuleNamePatch.php

```diff
@@ -50,7 +57,14 @@ public function rules()
                 'libs/libphp\$PHP_VERSION.$1'
             );
 
-        $rules[] = RegExpPatchRule::files(array('Makefile.global'))
+        // For php 7.4.0 and newer, patch 'build/Makefile.global', or patch 'Makefile.global'
```

Comment:
This comment is redundant. This is exactly what the code below reads like. The comment should be about why we're doing this, possibly with external references, not what we're doing. Please remove or rework.

Reply:
Okay.

Reply:
Done